medical_his/README.rst

```diff
@@ -0,0 +1,188 @@
+[![Build Status](https://travis-ci.org/OCA/vertical-medical.svg?branch=8.0)](https://travis-ci.org/OCA/vertical-medical)
```

Comment:
Same comment as above. The descriptions needs to be rewritten to properly describe specifically each module. Since